src/ngraph_emitter.cc

```diff
@@ -1922,7 +2016,9 @@ void Emitter::UnsupportedOps() {
     const auto out_shape = TShape_to_NShape(node->shape_);
     auto data = makeConstant(node->inputs_[0], "0");
     auto filter = makeConstant(node->inputs_[1], "0");
-    auto conv = create_deconvolution(data, filter, out_shape, node->orig_node_);
+    const auto& param = nnvm::get<mxnet::op::DeconvolutionParam>(
+        node->orig_node_->attrs.parsed);
+    auto conv = create_deconvolution(data, filter, out_shape, param);
```

Comment:
you may want to take a ref ?

Reply:
I'm not sure I understand, could you please clarify?

Reply:
never mind, i guess this is a temporary. was trying to see if we can avoid copying shared_ptrs.